testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -39,6 +43,58 @@ def test_transform_StoR_pass(coord_dtype):
     assert_equal(original_r, test_r)
 
 
+def test_capped_distance_noresults():
+    point1 = np.array([0.1, 0.1, 0.1], dtype=np.float32)
+    point2 = np.array([0.95, 0.1, 0.1], dtype=np.float32)
+
+    pairs, distances = mda.lib.distances.capped_distance(point1,
+                                                        point2,
+                                                        max_cutoff=0.2)
+
+    assert_equal(len(pairs), 0)
+
+
+boxes_1 = (np.array([1, 2, 3, 90, 90, 90], dtype=np.float32),  # ortho
+           np.array([1, 2, 3, 30, 45, 60], dtype=np.float32),  # tri_box
+           triclinic_vectors(  # tri_vecs
+           np.array([1, 2, 3, 90, 90, 45], dtype=np.float32)),
+           np.array([[0.5, 0.9, 1.9],  # tri_vecs_bad
+                     [2.0, 0.4, 0.1],
+                     [0.0, 0.6, 0.5]], dtype=np.float32))
+
+query_1 = (np.array([0.1, 0.1, 0.1], dtype=np.float32),
+           np.array([[0.1, 0.1, 0.1],
+                     [0.2, 0.1, 0.1]], dtype=np.float32))
+
+method_1 = ('bruteforce', 'pkdtree')
+
+np.random.seed(90003)
+points = (np.random.uniform(low=0, high=1.0,
+                        size=(100, 3))*(boxes_1[0][:3])).astype(np.float32)
+
+
+@pytest.mark.parametrize('box, query , method',
```

Comment:
I think if you stack the decorators, you don't need the product call, it does it itself, ie

```python
@pytest.mark.parametrize('box', ...)
@pytest.mark.parametrize('query', ...)
@pytest.mark.parametrize('method', ...)
def test_thing(box, query, method):

```